config/jobs/kubernetes/sig-k8s-infra/trusted/sig-k8s-infra-apps-periodics.yaml

```diff
@@ -1,7 +1,7 @@
 periodics:
 - name: periodic-k8sio-deploy-app-codesearch
   cluster: k8s-infra-prow-build-trusted
-  interval: 24h
+  cron: "0 10 * * 1,4" # At 10:00 UTC on Monday and Thursday.
```

Comment:
we have a lot of activities on our repos. I don't think it's gonna be enough to cover all PR merged. Ultimately every 6-12h is needed.